service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -36,8 +36,24 @@
 
     private static final String AUTH_TOKEN = "Bearer token";
     private static final String USER_ID = "1";
+    private static final String ORDERS_AND_DIRECTIONS = "ordersAndDirections";
+    private static final String CAFCASS = "cafcass";
+    private static final String SUBMITTED_CALLBACK_URL = "/callback/case-submission/submitted";
+    private static final String AUTHORIZATION = "authorization";
+    private static final String USER_ID_STR = "user-id";
+
     private static final ObjectMapper MAPPER = new ObjectMapper();
 
+    private static final Map<String, Object> expectedSimilarParameters = ImmutableMap.<String, Object>builder()
+        .put("localAuthority", "Example Local Authority")
```

Comment:
Should we call this expectedParameters instead? We don't want similar we want the exact same?